.env.example

```
@@ -16,9 +16,24 @@ DATABASE_URL=
 # -----------------------------------------------------------------------------
-# Anthropic API key — powers every agent in the pipeline.
+# OpenAI/Codex is the preferred LLM provider. The code checks CODEX_API_KEY first,
+# then OPENAI_API_KEY, then falls back to Anthropic only if neither is set.
+# Optional kill switch: openai | anthropic. Leave blank for key-based priority.
+LLM_PROVIDER=
+# OpenAI's official API key environment variable is OPENAI_API_KEY; CODEX_API_KEY
+# is accepted as a project-local alias if you want to keep Codex credentials separate.
+CODEX_API_KEY=
+OPENAI_API_KEY=
+# Defaults to gpt-5.2-codex when an OpenAI key is present. Override if needed.
+CODEX_MODEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CODEX_MODEL key should go before the LLM_PROVIDER key

(UnorderedKey)